openstack/containerinfra/v1/clustertemplates/results.go

```diff
@@ -58,3 +65,17 @@ type ClusterTemplate struct {
 	UserID              string             `json:"user_id"`
 	VolumeDriver        string             `json:"volume_driver"`
 }
+
+func (r DeleteResult) Extract() (string, error) {
```

Comment:
This is a unique way of handling retrieval of the Request ID. Is this really needed or can this be handled with the standard implementation of:

```go
res := clustertemplates.Delete(client, id)
if res.Err != nil {
  panic(res.Err)
}

requestID := res.Header["X-Openstack-Request-Id"][0]
```

Reply:
Agree, should keep it consistent. Will fix.